src/Common/src/Interop/Windows/mincore/Interop.ConsoleScreenBufferInfo.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class mincore
+    {
+        [StructLayoutAttribute(LayoutKind.Sequential)]
+        internal struct CONSOLE_SCREEN_BUFFER_INFO
```

Comment:
ConsolePAL.Windows.cs references these structs as well and so I decoupled them from the pinvoke but grouped the structs together because for now they all get used together :).